src/Datadog.Trace.AspNet/Datadog.Trace.AspNet.csproj

```diff
@@ -0,0 +1,20 @@
+﻿<Project Sdk="Microsoft.NET.Sdk">
+
+  <PropertyGroup>
+    <TargetFrameworks>net45</TargetFrameworks>
+
+    <!-- NuGet -->
+    <Version>1.0</Version>
```

Comment:
Can you modify `tools/SynchronizeVersions/Program.cs` to include `src/Datadog.Trace.AspNet/Datadog.Trace.AspNet.csproj` and run it to verify it updates this to `1.10.0`?

Reply:
Added to SyncVersions in 0abd2c3921dca41cb882cf1ea43d6fd89a905f0e and ran it locally.